addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -549,7 +580,7 @@ export class MessageList extends Component {
             // Automatically scroll to new received messages only when the list is
             // currently fully scrolled.
             const hasAutoScrollOnMessageReceived = (order === 'asc')
-                ? scrollTop >= this._getScrollableElement().scrollHeight - this._getScrollableElement().clientHeight - margin
+                ? this.threadView.isAtEnd
                 : scrollTop <= margin;
```

Comment:
`hasAutoScrollOnMessageReceived` can only be a compute on the model now depending on `isAtEnd`, as this here doesn't use the event at all now.

Reply:
This also depend of the scrollTop that rely on a props. I don't see how I could compute with that.

Reply:
Save "is at top", save "is at bottom", and then you have all you need in the model.